translations/locales_startup.yaml

```diff
@@ -120,16 +120,16 @@ POPUP_CUSTOM_PROJECT_BUTTON_OK:
   ja: "了解"
   fr: "Ok"
   nl: ""
-  zh_TW: ""
+  zh_TW: "確定"
   pl_pl: ""
   de: ""
   pt_br: ""
 POPUP_CUSTOM_PROJECT_TOOLTIP_X:
   en: "Width in pixels for the project."
-  ja: "プロジェクトの幅（ピクセル）
+  ja: "プロジェクトの幅（ピクセル）"
```

Comment:
Thanks about that, accidentally deleted this whilst approving the French translations.